lighthouse-cli/performance-experiment/server.js

```diff
@@ -20,30 +20,35 @@
  * @fileoverview Server script for Project Performance Experiment.
  *
  * Functionality:
- *   Host and open report page.
+ *    Host experiment.
+ *    Report can be access via URL http://localhost:[PORT]/
+ *    Rerun data can be access via URL http://localhost:[PORT]/rerun.
+ *      This will rerun lighthouse with same parameters and rerun results in JSON format
  */
 
 const http = require('http');
 const parse = require('url').parse;
-const path = require('path');
 const opn = require('opn');
 const log = require('../../lighthouse-core/lib/log');
-const ReportGenerator = require('../../lighthouse-core/report/report-generator');
+const reportGenerator = new (require('../../lighthouse-core/report/report-generator'))();
+const lighthouse = require('../../lighthouse-core');
+const perfOnlyConfig = require('../../lighthouse-core/config/perf.json');
+
+const hostedExperiments = [];
```

Comment:
Right now the server stores all the results / configs here (in the memory). I'm aware of the potential memory issue. After we have a front end that can compare performance data, we can directly forward the results to that front end page and don't need to store them in memory (at least not in cli).